lib/install/validate/validator.go

```diff
@@ -156,6 +157,30 @@ func NewValidator(ctx context.Context, input *data.Data) (*Validator, error) {
 	return v, nil
 }
 
+var schemeMatch = regexp.MustCompile(`^\w+://`)
+
+// Starting from GO 1.8 the URL parser does not
```

Comment:
Doc should start with the function name (to avoid go-lint warnings).
s/GO/Go/
s/Schema/Scheme/g